### ISSUE TITLE
[Discussion] Zelda Pause-Menu Framebuffer Issues under certain conditions

### ISSUE THREAD

**Issue by Testsr** [libretro/mupen64plus-libretro-nx#172]
#49 
Attached is a zip of game footage showing the bug.
Rom details:
Legend of Zelda, The - Ocarina of Time (U) (V1.2) [!].z64
CRC-32:32120c23
MD5:57a9719ad547c516342e1a15d5c28c3d
SHA256: 49acd3885f13b0730119b78fb970911cc8aba614fe383368015c21565983368d

**Comment by BParks21** [comment]
I have this issue as well. On unmodified roms. This isn't my only rom but this is an unmodified version of the game, a raw dump it should be added to the database. 

https://datomatic.no-intro.org/index.php?page=show_record&s=24&n=0376
Format:	BigEndian
MD5: 5bd1fe107bf8106b2ab6650abecd54d6
CRC64: FC96461102B8F02E
SHA-1: AD69C91157F6705E8AB06C79FE08AAD47BB57BA7

**Comment by m4xw** [comment]
Someone give this a shot: [mupen64plus_next_libretro.zip](https://github.com/libretro/mupen64plus-libretro-nx/files/4609187/mupen64plus_next_libretro.zip)

**Comment by m4xw** [comment]
In case you get "failed to open libretro core": https://git.m4xw.net/libretro/cores/mupen64plus-libretro-nx/-/jobs/7728/artifacts/browse

**Comment by BParks21** [comment]
@m4xw Fixed it for me. I can't reproduce it.

**Comment by BParks21** [comment]
Yes I'm very happy this is fixed. Thank for putting the time in to figure this out.

**Comment by m4xw** [comment]
@BParks21 Theres a few hold ups currently for the release, mostly upstream issues.
We currently try to nail down the oot freeze on upstream, but yea it will be in the next release.
Probably few days at least.

**Comment by BParks21** [comment]
Cool.
